File: src/prog_langs/python/read.py
import MySQLdb

# 接続するための情報
conn = MySQLdb.connect(
	user   = 'guest',
	passwd = 'DK4XMDVATL2N8448CMF9',
	host   = 'localhost',
	db     = 'python'
)

# カーソルを指定、python独自らしい
cur = conn.cursor()
# テーブルtoyのデータを取得するsql文を格納
sql = "SELECT * FROM toy"
# sql文を実行
cur.execute(sql)
# 取得したデータをresultに格納
results = cur.fetchall()
# resultを表示
print(results)
# resultをfor文で回す
for r in results:
    print(r)

# 実行結果
# ((1, 'bear'),)
# (1, 'bear')

# カーソルを終了
cur.close
# db接続を終了
conn.close

File: src/prog_langs/php/read.php
<?php
	$db = [
		'host' => 'localhost',
		'name' => 'guest',
		'pass' => 'DK4XMDVATL2N8448CMF9',
		'db'   => 'php'
	];

	$mysqli = new mysqli("$db[host]","$db[name]","$db[pass]","$db[db]");// dbに接続

	if( $mysqli->connect_errno ) {
		echo $mysqli->connect_errno . ' : ' . $mysqli->connect_error;
		return false;
	}

	mysqli_set_charset($mysqli, 'utf8');// 文字コードをセット
	$sql = "SELECT * FROM toy ORDER BY id DESC";// toyの内容をid降順で取得する文
	$res = mysqli_query($mysqli, $sql);// 実際に取得

	$mysqli->close();// dbを閉じる
	var_dump(mysqli_fetch_assoc($res));// 取得したものをvar_dumpで表示

	// 実行結果
	// array(2) {
	//   ["id"]=>
	//   string(1) "1"
	//   ["name"]=>
	//   string(4) "bear"
	// }
?>